components/Button.tsx

```diff
@@ -26,7 +26,7 @@ export default function Button({
       break
     case "tertiary":
       buttonStyles =
-        "bg-neutral-50 text-secondary-blue-700 hover:bg-neutral-100"
+        "bg-white text-secondary-blue-700 hover:bg-neutral-100 hover:text-white"
```

Comment:
What's the contrast like on these hover styles?  `neutral-100` feels pretty light when compared with white text... Do you have an example of this on the site? I might be misunderstanding